### DISCUSSION TITLE
Update current tarif sensor on the hour

### DISCUSSION THREAD

**Discussion by berendhaan** [fsaris/home-assistant-zonneplan-one#68] [Ideas]
The current sensor is not updated on exactly the hour. This is not very helpful when creating an automation to, for example, switch off or switch on appliances based on if the price is positive or negative. 

Because of this behavior my automation ran 5 minutes late, being Dutch this is a big issue lol.

**Comment by fsaris** [comment]
Are the updated prices exactly on hour change visible in the app?

**Reply by berendhaan** [reply]
Maybe a bit later. But HA shows .27 at the moment but the app shows .28. 
Edit: On the 4th minute it updated.

**Comment by berendhaan** [comment]
I've been watching the behavior today, and it looks like it updates on the hour on at least 3 occasions.
I assume you just poll their API to get the current price, so it could be a caching 'issue' from their end.

A more solid method would be using the forecast data to fill the current value of the sensor.
